modules/blog/templates/front/block.featured-blogs.tpl

```diff
@@ -0,0 +1,28 @@
+{if isset($blog_blocks_data.featured)}
+    <div class="new-blog-posts">
+        {foreach $blog_blocks_data.featured as $one_blog_entry}
+            <div class="media">
+                {if $one_blog_entry.image}
```

Comment:
it's better to use `!empty` as it will not generate an error when customers might delete this field.